cypress/e2e/environment.cy.js

```diff
@@ -0,0 +1,389 @@
+  });
+
+  it("1.4 Edit created environment", function () {
+    //Fill The form and submit
+    cy.visit("/console/environment/create");
+    createEnv(4, true, true);
+    cy.get("button").contains("Submit").click();
+    //Go to settings
+    cy.get('[aria-label="Settings actions"]').click();
+    cy.get('[aria-label="Name-value"]').should("contain", testName(4));
+    cy.wrap(parameterTypes).each((parameter) => {
+      if (parameter.name !== "Repository" && parameter.name !== "Branch") {
+        cy.get('[aria-label="' + parameter.name + '-toggle-edit"]').click({
+          force: true,
+        });
+      }
+
+      switch (parameter.type) {
+        case "text":
+          cy.get('[aria-label="' + parameter.name + '-input"]')
+            .clear()
+            .type("New Value " + parameter.name);
+          break;
+        case "icon":
+          cy.get(iconInput).attachFile(icon);
+          break;
+        case "typeahead":
+          cy.get('[aria-label="Project Name-typeahead"]')
+            .clear()
+            .type("New Value " + parameter.name);
+
+          cy.get('[aria-label="Project Name-select-input"]').then(($list) => {
+            if (
+              $list.find(
+                (element) =>
+                  element.textContent === `Create "New Value ${parameter.name}"`
+              ).length > 0
+            ) {
+              cy.get("button")
+                .contains(`Create "New Value ${parameter.name}"`)
+                .click();
+            } else {
+              cy.get("button").contains(`New Value ${parameter.name}`).click();
+            }
+          });
+
+          break;
+        case "joined":
+          cy.get('[aria-label="Repository Settings-toggle-edit"]').click();
+          cy.get(
+            parameter.name === "Branch"
+              ? '[aria-label="repo_branch-input"]'
+              : '[aria-label="repo_url-input"]'
+          )
+            .clear()
+            .type("New Value " + parameter.name);
+          break;
+      }
+      if (parameter.name !== "Repository" && parameter.name !== "Branch") {
+        cy.get('[aria-label="' + parameter.name + '-submit-edit"]').click();
+        cy.get('[aria-label="' + parameter.name + '-value"]').should(
+          "contain",
+          "New Value " + parameter.name
+        );
+      } else {
+        cy.get('[aria-label="Repository Settings-submit-edit"]').click();
+        if (parameter.name === "Repository") {
+          cy.get('[aria-label="repo_url-value"]').should(
+            "contain",
+            "New Value " + parameter.name
+          );
+        } else {
+          cy.get('[aria-label="repo_branch-value"]').should(
+            "contain",
+            "New Value " + parameter.name
+          );
+        }
+      }
+    });
+    deleteEnv(4);
+  });
+
+  it("1.5 Clear environment", function () {
+    //Fill The form and submit
+    cy.visit("/console/");
+    cy.get('[aria-label="Environment card"]').contains("test").click();
```

Comment:
I noticed that it will pick any environment that 'contains' the word test, the idea was to use the frontend-lsm one, in the test I'm writing I had to use the project label to be sure it picked the right one.

Reply:
You are right, I changed it